packages/cypress/tests/cypress/e2e/custom-downloads-directory.cy.ts

```diff
@@ -1,9 +1,11 @@
-context('using Cypress.config', () => {
-  it('Downloads archives to the user-specified folder', () => {
-    cy.visit('/viewports');
+it(
+  'downloads archives to the user-specified folder',
+  { env: { disableAutoSnapshot: true } },
```

Comment:
Can a comment clarify why we're disabling auto snapshots here? Just since it's not what the test is testing, I was wondering why the config was needed.

Reply:
Added a comment in f74bde795646780bdbf4fe76b331102439c7ab5d